src/restorepackages.bat

```diff
@@ -27,16 +28,17 @@ set NugetConfig=%ConfigDir%\dynamo-nuget.config
 REM 2. replace "LatestBeta" strings in packages-template.aget with the latest
 REM    pre-release version of DynamoVisualProgramming.Core package,
 REM    and save the replaced file as packages.aget
-set versionQuery=%NugetExe% list DynamoVisualProgramming.Core -prerelease -config "%NugetConfig%"
-for /F "tokens=2 delims= " %%F in ( '%versionQuery%' ) do (set LatestBeta=%%F)
-echo Latest pre-release version of "DynamoVisualProgramming.Core" package is "%LatestBeta%"
-
+set versionQuery=list DynamoVisualProgramming.Core -prerelease -config "%NugetConfig%"
+for /F "tokens=2 delims= " %%F in ( 'call "%NugetExe%" %versionQuery%' ) do (
+    set LatestBeta=%%F
+)
+echo Latest pre-release version of "DynamoVisualProgramming.Core" package is %LatestBeta%
```

Comment:
Great, now since we are all here. @ksobon would you explain this change a bit and share your wisdom here with the Dynamo Team 😃

Reply:
@ellensi fixed this before, but I pulled the repo fresh this past weekend and was getting the same errors as before. I am keeping the repo on my google drive so the path to source file `%~dp0` needs to be able to handle spaces since mine looks like this: C:\Users\user\Google Drive\DynamoRevit....I didn't change much, but rather re-instated whatever @ellensi did before which was adding "" marks around variables inside of the loop so `"%NugetExe%"`. That was getting broken because we use `delims = "` which I believe was delineating the path variable at a space and causing issues....

Reply:
@ksobon Got you. I looked at the history of this file and seems the fix did get in for Revit2017 branch but not the others, after we merging this. Would you mind cherry-pick this change to Revit 2019 branches as well? Let me know if you would like to do it. LGTM